integration-tests/suites/base.go

```diff
@@ -406,7 +406,7 @@ func (s *IntegrationTestSuiteBase) RunImageWithJSONLabels() {
 func (s *IntegrationTestSuiteBase) StartContainerStats() {
 	name := "container-stats"
 	image := common.QaImage("quay.io/rhacs-eng/collector-performance", "stats")
-	args := []string{name, "-v", "/var/run/docker.sock:/var/run/docker.sock", image}
+	args := []string{name, "-v", common.RuntimeSocket + ":/var/run/docker.sock", image}
```

Comment:
How does this work? We mount the podman socket to `/var/run/docker.sock` in the container and it just works?

Reply:
![image](https://user-images.githubusercontent.com/20206636/223436465-d9674f02-d668-4f7c-953f-778d5ab3f88e.png)